apps/transport/lib/transport_web/live/on_demand_validation_select_live.ex

```diff
@@ -57,7 +63,10 @@ defmodule TransportWeb.Live.OnDemandValidationSelectLive do
 
   defp new_changeset(socket, params) do
     changeset = socket_value(socket, :changeset)
-    changeset |> Ecto.Changeset.change(params |> Map.new(fn {k, v} -> {String.to_atom(k), v} end))
+    allowed_fields = Map.keys(changeset.data())
+
+    changeset
+    |> Ecto.Changeset.change(params |> Map.new(fn {k, v} -> {String.to_atom(k), v} end) |> Map.take(allowed_fields))
```

Comment:
Va corriger https://sentry.io/organizations/transport-data-gouv-fr/issues/3081347668/?project=6197733&query=is%3Aunresolved

Reply:
tu dois pouvoir utiliser `string_to_existing_atom`, il est en général pas recommandé de convertir des strings en atom au runtime, parce que les atom ne sont jamais effacés. http://www.moxleystratton.com/when-to-use-elixirs-string-to-atom/

Aussi, je vais regarder demain par curiosité, je suis étonné qu'on doive passer par cette gymnastique pour filtrer les clés autorisées.

Reply:
https://github.com/etalab/transport-site/pull/2204/commits/690582c62ae5d0f440e1a679ff563e48050f15c9 does the trick